docs/fundamentals/dashboard/explore.md

```diff
@@ -429,6 +429,22 @@ The following shortcuts are available:
 - <kbd>?</kbd>: Got to **Help**.
 - <kbd>Shift</kbd> + <kbd>s</kbd>: Go to **<u>S</u>ettings**.
 
+## Interaction prompts
+
+Some resources or commands might prompt you for values when using the .NET Aspire dashboard. This interactive functionality is powered by the [interaction service](../../extensibility/interaction-service.md), which allows integrations to display notifications or to request input from users when needed.
+
+For example, Azure resources that are missing required configuration might prompt you for configuration values when the dashboard starts or when you interact with those resources. These prompts help ensure that resources are properly configured and can function correctly within your .NET Aspire application.
+
+In the dashboard, interaction prompts appear as:
+
+- Input dialogs for missing configuration values
+- Confirmation dialogs for important actions
+- Notification messages with details about resource status
```

Comment:
Per our style guide, bulleted lists should form complete sentences and use proper punctuation.